playbooks/files/dynamic_inventory.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2024 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
```

Comment:
Thanks for adding this!

Reply:
FYI... I updated this in the MSD branch

https://github.com/netascode/ansible-dc-vxlan/pull/249/commits/58dd0f0e03b54d933bb859b8fab0a1619272fdaa